weatherAPI/libraries/response.go

```diff
@@ -0,0 +1,152 @@
+package libraries
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/astaxie/beego/httplib"
+	"github.com/astaxie/beego/logs"
+)
+
+func GetResponse(city, country string) map[string]interface{} {
+	var jsonResponse map[string]interface{}
+	response := make(map[string]interface{})
+
+	uri := fmt.Sprintf("http://api.openweathermap.org/data/2.5/weather?q=%v,%v&appid=1508a9a4840a5574c822d70ca2132032", city, country)
```

Comment:
The url can be split in different fragments, like `base_ulr`, `appid` and so on.
Those values can be moved to the `app.conf` file and and inally the url can be composed here.